src/vstt/vis.py

```diff
@@ -98,17 +99,21 @@ def make_target_labels(
 
 
 def update_target_colors(
-    targets: ElementArrayStim, show_inactive_targets: bool, index: int | None = None
+    targets: ElementArrayStim,
+    show_inactive_targets: bool,
+    index: int | None = None,
+    tm: TrialManager | None = None,
```

Comment:
we don't need to pass the whole TrialManager here
```suggestion
    green_target_index: int | None = None,
```